client_connection/manager.go

```diff
@@ -14,7 +14,9 @@ import (
 
 type DialogEstablisherFactory func(identity identity.Identity) communication.DialogEstablisher
 
-type VpnClientFactory func(vpnSession session.SessionDto) (openvpn.Client, error)
+type VpnClientFactory func(vpnSession session.SessionDto, identity identity.Identity) (openvpn.Client, error)
+
+type SignerFactory func(identity identity.Identity) identity.Signer
```

Comment:
`identity` is package name, lets not use it for variables.
Use `id`, `providerId`, `customerId`, `myId`, `signerId`

Reply:
Will fix that.